wrs/ceph-init-wrapper.sh

```diff
@@ -162,8 +177,21 @@ log_and_kill_hung_procs ()
     done
 }
 
+
 status ()
 {
+    if [[ "$system_type" == "All-in-one" ]] && [[ "$system_mode" != "simplex" ]] && [[ "$1" == "osd" ]]
+    then
+        timeout 20 ceph -s
```

Comment:
20 should be a variable above at the top of the file. I assume that we want a smaller timeout here instead of using GET_STATUS_TIMEOUT=120 as this blocks a swact for 20s? If so, is there a better way to determine a swat is underway rather than a 'ceph -s' timeout?

Reply:
The change should not delay swact as this is not intended for ceph-mon, it's intendend for ceph-osd checks. There is a probability that SM's periodic thread checks OSD status during swact and we want to avoid false errors of ceph osd processes while ceph-mon is re-initializing (ceph osd runs on both controllers and is also managed by SM).